enhancements/network/persistent-ips-sdn-secondary-networks.md

```diff
@@ -403,7 +403,46 @@ TODO
 
 ### Integrate the `PersistentIP` CRD with OVN-Kubernetes ip pool
 
-TODO
+We plan on integrating the proposed `PersistentIP`s CRD with the existing
+OVN-Kubernetes IP pool allocation mechanism by reacting to two different
+events: ovnkube-master start, and `PersistentIP` CR delete.
+
+The updates to the `PersistentIP` CR will not be supported, and since they will
+be provisioned as part of the regular pod triggered CNI ADD operation, it is
+not necessary to react to provisioned `PersistentIP`s.
+
+#### How to integrate the new CRD with the OVN-K IP pool on ovnkube-master start
+
+When the ovnkube-master starts, it will - for every pod in the system - invoke
+the `AllocateIPs` function from the allocator. We will need to also read all
+the `PersistentIP`s from the informer cache, and invoke the `AllocateIPs`
+function for each of the persisted IPs, as depicted in the following diagram.
```

Comment:
The first sence is what it's doing already we need to put here just the second sentence

```
At ovnkube-control-plane start, it will read all the PersisentIPs
from the informer cache, and invoke the `AllocateIPs`
function for each of the persisted IPs, as depicted in the following diagram.
```